devops/resources/export-slow-query-logs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd $WORKSPACE/configuration/util/jenkins/export_slow_logs
+
+pip install -r requirements.txt
+. ../assume-role.sh
+
+# Assume the role
+set +x
+assume-role ${ROLE_ARN}
+set -x
+
+python export_slow_query_logs.py  --username ${USERNAME} --password ${PASSWORD}
```

Comment:
Please load password from an ENV, passing it in this way will expose it to anyone who runs ps aux on the machine(possibly other jobs)